k8s/templates/dispatch_api_template.yml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sample_deployment
+  namespace: {{ deployment.namespace }}
+  labels:
+    application: ml-model-ci-deployment
+spec:
+  replicas: {{ deployment.replicas }}
+  minReadySeconds: 15
+  selector:
+    matchLabels:
+        application: ml-model-ci-deployment
+  template:
+    metadata:
+      namespace: {{ deployment.namespace }}
+      labels:
+        application: ml-model-ci-deployment
+    spec:
+      initContainers:
```

Comment:
So, now this PR only supports model retrieval and dispatch APIs?

Reply:
1. Model retreval part in deployment script is different from modelci retrieval api.
modelci retrieval api is not suitable for k8s deployment (local storage, stateful service e.g. mongoDB)
therefore I assume use has remote storage of compiled models.
2. This PR integrated with dispatch part. Targeting at providing a deployment script for deploying cloud service with model.